openshift/secrets-template.yaml

```diff
@@ -124,7 +124,59 @@ objects:
   data:
     token: ${SNYK_TOKEN}
     iss: ${SNYK_ISS}
-
+    encryption_key_for_snyk_token: ${ENCRYPTION_KEY_FOR_SNYK_TOKEN}
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: developer-analytics-audit-report-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket: 
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: cvae-npm-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: hpf-maven-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: hpf-pypi-insights-s3
+  type: Opaque
+  data:
+    aws_secret_access_key: ${AWS_SECRET_ACCESS_KEY}
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID}
+    bucket:
```

Comment:
It good you have added all missing secrets for insight and audit components, but can set the bucket name using DEPLOYMENT_PREFIX as below, as a rule developers are following naming conventions for these buckets in S3. So this will work for most of the developers directly.

bucket: ${DEPLOYMENT_PREFIX}-hpf-pypi-insights

Example :: dhpatel-hpf-pypi-insights

Reply:
Is this really required? Arent these local to the project?

Reply:
There are local to the deployment, for each deployment Dev / Stg / Prod we follow the same naming convention. We can always override this value. You can see prod buckets are like 'prod-hpf-pypi-insights', so thought to keep default that will work for most of the cases.

Reply:
done